Makefile

```diff
@@ -116,6 +116,19 @@ cert-mgr: ## Install the certification manager
 	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/$(CERT_MGR_VERSION)/cert-manager.yaml
 	kubectl wait --for=condition=Available --namespace=cert-manager deployment/cert-manager-webhook --timeout=60s
 
+uninstall: ## Uninstall all rukpak-related content from the cluster
```

Comment:
Should this also delete the webhooks we install in the current `deploy`/`run` target?